insalan/tournament/views/team.py

```diff
@@ -179,6 +179,33 @@ def patch(self, request, *args, **kwargs):
                     MailManager.get_mailer(EMAIL_AUTH["tournament"]["from"]).send_kick_mail(substitute.as_user(), team.name)
                     substitute.delete()
 
+        if "seat_slot" in data:
+            try:
+                seat_slot = SeatSlot.objects.get(id=data["seat_slot"])
+            except SeatSlot.DoesNotExist:
+                return Response({
+                    "seat_slot": _("Slot invalide.")
+                }, status=status.HTTP_400_BAD_REQUEST)
+
+            if seat_slot.tournament != team.tournament:
+                return Response({
+                    "seat_slot": _("Slot appartient à un autre tournoi.")
+                }, status=status.HTTP_400_BAD_REQUEST)
+
+            if hasattr(seat_slot, "team") and seat_slot.team != team:
```

Comment:
You should rather check the ID because this function is editing some fields and the save is only at the end. You can't predict if python comparison between objects will still work.